src/ppx/grfx/grfx_swapchain.cpp

```diff
@@ -177,23 +184,66 @@ void Swapchain::DestroyDepthImages()
     mDepthImages.clear();
 }
 
+Result Swapchain::CreateRenderTargets()
+{
+    uint32_t imageCount = CountU32(mColorImages);
+    PPX_ASSERT_MSG((imageCount > 0), "No color images found for swapchain renderpasses");
+    for (size_t i = 0; i < imageCount; ++i) {
+        auto                             imagePtr      = mColorImages[i];
+        grfx::RenderTargetViewCreateInfo rtvCreateInfo = grfx::RenderTargetViewCreateInfo::GuessFromImage(imagePtr);
+        rtvCreateInfo.loadOp                           = ppx::grfx::ATTACHMENT_LOAD_OP_CLEAR;
+        rtvCreateInfo.ownership                        = grfx::OWNERSHIP_RESTRICTED;
+        grfx::RenderTargetViewPtr rtv;
+        Result                    ppxres = GetDevice()->CreateRenderTargetView(&rtvCreateInfo, &rtv);
```

Comment:
nit: Would you mind adding a space between the createInfo fill out and the call to Create*  to make things a bit easier to visually parse?

Reply:
Done!